api/api.go

```diff
@@ -134,6 +135,9 @@ func NewAPI(conf *APIConfig) (*API, error) {
 		IdleTimeout:  time.Second * 60,
 		Handler:      mux,
 		TLSConfig:    t,
+		ErrorLog: logger.StandardLogger(&hclog.StandardLoggerOptions{
```

Comment:
See my comment replying to John's comment about the log level

Reply:
Yeah, based on that description, it sounds like there can be errors that are logged but do need intervention on the server side. I'll update it to always log with WARN!

Reply:
```
2021-11-12T13:36:59.644-0600 [WARN]  api: http: TLS handshake error from [::1]:62024: EOF
```